Exec/SCIENCE/ecsn/GNUmakefile

```diff
@@ -20,7 +20,7 @@ MICROPHYSICS_HOME ?= ../../../../Microphysics
 # If not we use the version in $(MICROPHYSICS_HOME)/Microphysics/(EOS/conductivity/networks)
 EOS_DIR          := helmholtz
 CONDUCTIVITY_DIR := stellar
-NETWORK_DIR      := ECSN
+NETWORK_DIR      := ECSN_Donald
 
```

Comment:
is this really the new network name?

Reply:
Yes, ECSN_Donald is the newest network. I can modify the name to ECSN. and delete the old version.

Reply:
yes, please just update ECSN without renaming.

Reply:
What was the difference between ECSN and ECSN_Donald?

Reply:
I just check. In my local machine, ECSN_Donald is modified by you but ECSN is not. But they are the same in github.

Reply:
Oh okay, that was probably back when we had an old and new version of the ECSN network before and after the changes to the energy generation. But now those changes are in the development branch of the Microphysics repository for the ECSN network.